t/BikeSharing/IndegoPHL.t

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env perl
+
+use strict;
+use warnings;
+use Test::More;
+use DDG::Test::Spice;
+
+ddg_spice_test(
+    [qw( DDG::Spice::BikeSharing::IndegoPHL )],
+    'philly indego' => test_spice(
+        '/js/spice/bike_sharing/indego_phl/philly',
+        call_type => 'include',
+        caller => 'DDG::Spice::BikeSharing::IndegoPHL',
+    is_cached => 1,
```

Comment:
Whoops, your test is failing because you disabled caching in your .pm file. Once you re-enable `spice is_cached` tests will pass.